consultation_analyser/middleware.py

```diff
@@ -0,0 +1,17 @@
+from django.http import HttpResponseRedirect
+
+
+class SupportAppStaffRequiredMiddleware:
+    """Require staff users only for support app."""
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        response = self.get_response(request)
+        if request.path.startswith("/support/"):
+            if not request.user.is_authenticated:
+                return HttpResponseRedirect("/")
+            elif not request.user.is_staff:
+                return HttpResponseRedirect("/")
```

Comment:
Would it be better to github-style 404 in this case? I know it's slight status code misuse but it feels more secure than advertising that a page exists at this url?

Reply:
Now updated to 404 here.